README.md

```diff
@@ -1,285 +1,46 @@
-<a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Modify this file to match your project and remove sections that don't apply.
+# Bookstore
```

Comment:
# Optional
- Nice job for implementing the changes requested from the previous review. But you were not supposed to remove most of the new [updated ReadMe file sections](https://github.com/microverseinc/readme-template) such as the main title of the file on top, Microverse icon, and Table content ...
- Now your ReadMe resembles the old version, and your next move will likely be invalidated till you bring back the new ReadMe. All the best